plugins/synaptics-rmi/meson.build

```diff
@@ -1,3 +1,4 @@
+if get_option('gudev') and get_option('plugin_synaptics_rmi')
```

Comment:
this should follow the pattern of showing an error if plugin flag set but not gudev set

Reply:
Agree, fixed.